federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -42,9 +42,9 @@ spec:
           mountPath: /srv/kubernetes/
           readOnly: true
       - name: etcd
-        image: quay.io/coreos/etcd:v2.3.3
+        image: gcr.io/google_containers/etcd:3.0.14-alpha.1
         command:
-          - /etcd
+          - /usr/local/bin/etcd
```

Comment:
@madhusudancs , the change is done in federation/manifests/federation-apiserver-deployment.yaml also.
The federation-apiserver now is called with `--storage-backend=etcd2` [here](https://github.com/kubernetes/kubernetes/pull/39646/files#diff-292011a5f785a3eb1dfc13fe070ab8ffR17) is the change.